### ISSUE TITLE
ITU-R TF.460-6 cannot be fetched

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-itu#44]
This document is not found on the general ITU search site:
https://www.itu.int/search

Google clearly gives this for the same search:
https://www.itu.int/rec/R-REC-TF.460/en

It can be found here:
https://extranet.itu.int/brdocsearch/R-REC/Forms/folders_inforce.aspx

<img width="867" alt="Screen Shot 2020-12-07 at 4 29 37 PM" src="https://user-images.githubusercontent.com/11865/101327251-66c8db80-38a9-11eb-8cab-69e4ead8d5f9.png">

**Comment by ryzokuken** [comment]
@ronaldtse this still doesn't work, how could this be fixed?

**Comment by ronaldtse** [comment]
@ryzokuken sorry for the late reply!

In the meantime, can you try this:
```adoc
* [[[ITURTF460-6,(ITU-R TF.460-6)]]], _ITU-R TF.460-6. Standard-frequency and time-signal emissions_
```
(https://www.metanorma.org/author/topics/document-format/bibliography/#named-reference-tag-with-fully-specified-bibliographic-entry)

@andrew2net this is related to #55 and #45 , as ITU suggested we should just use the index.

**Comment by ryzokuken** [comment]
@ronaldtse no worries, and thanks a lot for the suggestion. This should work for me at the moment.

**Comment by ronaldtse** [comment]
@ryzokuken sorry again about this -- at least ITU gave us a way forward which we will implement!

**Comment by andrew2net** [comment]
@ronaldtse the ITU-R TF.460-6 document has R-REC-TF.460 identifier on its [page](https://extranet.itu.int/brdocsearch/R-REC/Forms/ITU%20Recommendation%20Folder/docsethomepage.aspx?ID=25991&FolderCTID=0x0120D52000F22870A5AFA1054181BAD87677D647C400051FC5857030E848B2F8C42DB4271329&List=0661b581-2413-4e84-bab2-77e6db27af7f&RootFolder=%2Fbrdocsearch%2FR%2DREC%2FR%2DREC%2DTF%2FR%2DREC%2DTF%2E460%2FR%2DREC%2DTF%2E460%2D6%2D200202%2DI&RecSrc=%2Fbrdocsearch%2FR%2DREC%2FR%2DREC%2DTF%2FR%2DREC%2DTF%2E460%2FR%2DREC%2DTF%2E460%2D6%2D200202%2DI). The documents stored in the realton-data-itu-r repo with filenames derived from their identifiers. So ITU-R TF.460-6 doesn't match to file and relaton-itu doesn't find it. Should we change the identifiers in relaton-data-itu-r to be the same as the reference? Or just change filenames (for example, "R-REC-TF.460" => "ITU-R_TF.460")?

**Comment by ronaldtse** [comment]
We could use the same pattern as reference. We just need to make sure that we can retrieve it…

**Comment by andrew2net** [comment]
fixed in version 1.8.2
